src/gfn/gym/set_addition.py

```diff
@@ -0,0 +1,72 @@
+from typing import Callable
+
+import torch
+
+from gfn.env import Actions, DiscreteEnv, DiscreteStates
+
+
+class SetAddition(DiscreteEnv):
+    """Append only MDP, similarly to what is described in Remark 8 of Shen et al. 2023
```

Comment:
Could you please add a one or two line explanation of this environment, e.g. states, actions... ?